packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1171,51 +1171,53 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
       get(sectionErrors[section.id][field.name], 'errors') ||
       this.getErrorForNestedField(section, field, sectionErrors)
 
-    return errorsOnField.length > 0
-      ? this.getFieldValueWithErrorMessage(section, field, errorsOnField[0])
-      : field.nestedFields && !Boolean(ignoreNestedFieldWrapping)
-      ? (
+    return errorsOnField.length > 0 ? (
+      this.getFieldValueWithErrorMessage(section, field, errorsOnField[0])
+    ) : field.nestedFields && !Boolean(ignoreNestedFieldWrapping) ? (
+      (
+        (data[section.id] &&
+          data[section.id][field.name] &&
+          (data[section.id][field.name] as IFormSectionData).value &&
+          field.nestedFields[
+            (data[section.id][field.name] as IFormSectionData).value as string
+          ]) ||
+        []
+      ).reduce((groupedValues, nestedField) => {
+        const errorsOnNestedField =
+          sectionErrors[section.id][field.name].nestedFields[
+            nestedField.name
+          ] || []
+        // Value of the parentField resembles with IFormData as a nested form
+        const nestedValue =
           (data[section.id] &&
             data[section.id][field.name] &&
-            (data[section.id][field.name] as IFormSectionData).value &&
-            field.nestedFields[
-              (data[section.id][field.name] as IFormSectionData).value as string
-            ]) ||
-          []
-        ).reduce((groupedValues, nestedField) => {
-          const errorsOnNestedField =
-            sectionErrors[section.id][field.name].nestedFields[
-              nestedField.name
-            ] || []
-          // Value of the parentField resembles with IFormData as a nested form
-          const nestedValue =
-            (data[section.id] &&
-              data[section.id][field.name] &&
-              renderValue(
-                data[section.id][field.name] as IFormData,
-                'nestedFields',
-                nestedField,
-                intl,
-                offlineCountryConfiguration,
-                language,
-                isOriginalData
-              )) ||
-            ''
-          return (
-            <>
-              {groupedValues}
-              {(errorsOnNestedField.length > 0 || nestedValue) && <br />}
-              {errorsOnNestedField.length > 0
-                ? this.getFieldValueWithErrorMessage(
-                    section,
-                    field,
-                    errorsOnNestedField[0]
-                  )
-                : nestedValue}
-            </>
-          )
-        }, <>{value}</>)
-      : value
```

Comment:
The essential change here, is that I wrapped the `: value` to be `<>{value}</>` to always return a ReactNode from the method.